<!doctype html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="landing page">
    <meta name="author" content="Tariq Khaleeq, Nikos Papadopoulos">
    <link rel="icon" href="./media/favicon.ico">

    <title>Newsletter</title>

    <!-- Bootstrap core CSS -->
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/css/bootstrap.min.css" integrity="sha384-Zug+QiDoJOrZ5t4lssLdxGhVrurbmBWopoEl+M6BdEfwnCJZtKxi1KgxUyJq13dy" crossorigin="anonymous">
    <link href="./style.css" rel="stylesheet" type="text/css" />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
    <style>
    body{
      font-family: 'Roboto';
    }
    </style>
</head>

<body>
  <nav class="navbar navbar-expand-md oxford_background top">
      <a class="navbar-brand text-white" href="main.html">RSG Germany</a>
      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse"
          aria-expanded="false" aria-label="Toggle navigation">
          <span class="navbar-toggler-icon"></span>
      </button>

      <div class="collapse navbar-collapse" id="navbarCollapse">
          <ul class="navbar-nav ml-auto">
              <li class="nav-item active">
                  <a class="nav-link disabled text-white" href="main.html">Home
                  </a>
              </li>
              <li class="nav-item">
                  <a class="nav-link disabled text-white" href="events.html">Events</a>
              </li>
              <li class="nav-item">
                  <a class="nav-link" href="about.html">About
                    <span class="sr-only">(current)</span>
                  </a>
              </li>
              <li class="nav-item">
                  <a class="nav-link disabled text-white" href="newsletter.html">Newsletter</a>
              </li>
              <li class="nav-item">
                  <a class="nav-link disabled text-white" href="contact.html">Contact</a>
              </li>
          </ul>
      </div>

  </nav>
  <!--TODO: add subscription button. Add Footer.
  Probably stylize background. Figure a way to make this dynamic other wise would require bruteforce to update this every month-->
  <div class="container">
    <div class="row">
      <div class="col-sm-12 text-center">
        <br>
        <h1>Newsletter</h1>
      </div>
      <div class="col-sm-12">
        <br>
        <h5>Introduction</h5>
      </div>
      <div class="col-sm-12">
        <p>We have selected for this month
          <ul>
            <li>Two articles related to the importance of miRNA in predicting disease</li>
            <li>One brief communication related to the personalized-precision medicine.</li>
          </ul>
         miRNA are small molecules that are important for several biological processes and are recurrent in several diseases.
         For this, we decided to make short resumes of different papers related to them.</p>
         <p>The first is a study about to the estimation of miRNA and how this can be confound in people doing sports, such as athletes.
         Then we passed to a more methodological paper describing how to build topological network describing miRNA in relation to transcription factors and diseases.</p>

         <p>Finally, the third and last one is a short communication about our field, computational biology, and the way it can be used already in the present but even more in the imminent future to understand and treat specific diseases. Have a look at them and enjoy our first newsletter of the year!!</p>
         <hr>
      </div>
    </div>
    <div class="row">
      <div class="col-sm-12">
        <br>
        <h5><b><span style="color: #6b1a1a;">miRNAs and sports: tracking training status and potentially confounding diagnoses</span> </b></h5>
      </div>
      <div class="col-sm-12">
        <p>The complete functionality of miRNA and specifically miRNA abundance in physiological processes is not fully clear nor understood and remains under investigation because miRNAs can be, for example, used in Alzheimer’s disease as biomarkers. The importance to use them as biomarkers for diseases, raised the question how is the effect of physical exercises on miRNA profiles and how can this be used later on to change disease diagnosis as well as prognosis. Investigating such questions requires computational biology since it is handled in a comprehensive way using network analysis. Network analysis is the manifested choice since more complex marker patterns are not only correlated to diseases.
        </p>

        <p>The upstream analysis includes standard computational biology procedures from the principal component analysis, hierarchical clustering with Euclidean distance as well as visualization. Downstream analysis has been carried out with a multi-step analysis, starting with mapping mature forms of them to precursors and calculating correlations of identified miRNAs to disease. Further steps use comprehensive in-house build computational biology algorithms and APIs by the authors: prediction of pathways with miEAA, KEGG pathway and gene ontology analysis with miRTargetLink, GeneTrail and GeneTrail2.</p>

        <p>This study represents a comprehensive miRNA atlas of athletes in their training mode and they found that miRNAs are important disease markers, e.g. fatigue and acute myocardial infarctions. At the same time it comes to the attention of the authors that the mode and the extent of training are important confounding factors for a miRNA based disease diagnosis.  Further investigation regarding the concrete influence of disease markers on medical basis remains to be seen.</p>

      </div>
      <div class="col-sm-12">
        <p><b>Reference:</b> <a href="https://translational-medicine.biomedcentral.com/articles/10.1186/s12967-016-0974-x">Hecksteden, A. et al. Erratum to: miRNAs and sports: tracking training status and potentially confounding diagnoses. Journal of Translational Medicine 14, (2016).</a></p>
      </div>
    </div>

    <div class="row">
      <div class="col-sm-12">
        <h5><span style="color: #057d13;"><b>TfmiR: a useful platform to analyze disease specific Transcription Factor and miRNA co-regulatory networks</b></span></h5>
      </div>
      <div class="col-sm-12">
        <p>
          Among the multiple genetic factors involved in the regulation of gene expression, transcription factors (TF) and microRNA (miRNA) are key players during fundamental biological process such as proliferation, differentiation, survival and apoptosis. Several diseases show phenotype typical of a lost in proliferation, aberrant differentiation and uncontrolled apoptosis. Given the importance of such molecules, it will be useful to provide an user-friendly platform in which the user is able to input a specific list of disease genes and receive as output the underlying regulatory network of the genes in relation to TF and miRNA.
        </p>
        <p>
          For this purpose TFmiR is a web server tool capable of integrates genome-wide transcriptional and post-transcriptional regulatory interactions and elucidate human diseases. The platform model a specific disease network considering: a list of genes up and down regulated for the disease and four different interaction such as TF/Disease Genes, TF/miRNA, miRNA/miRNA and miRNA/Disease Genes.
        </p>
        <p>
          How is this framework modeled from a statistical point of view? First the four different interactions list are downloaded from manually curated databases such as Pmmr and starBase and then pooled together based on the input disease genes to generate the entire combinatory regulatory network. The downstream analysis consist of 3 steps: 1) compute the regulatory sub-network of the four interaction types, 2) create the combined network of all interaction types and 3) the disease-specific network. For the first step an overlap between the input deregulated regulators (miRNA/genes) and the targets of the input deregulated regulators (genes/miRNAs) is obtained using the hypergeometric distribution test. To avoid false positives, a randomization test is conducted (n=1000). For the other two levels, TFmir test for each network basic topological features relevant to the disease-associated gene/miRNA by computing the overlap significance with the network nodes. A coverage ratio is computed considering the diseases-specific network and the nodes of the entire combined network.
        </p>
      </div>
      <div class="col-sm-12">
        <p><b>Reference:</b> <a href="https://academic.oup.com/nar/article/43/W1/W283/2467907">Methods</a></p>
      </div>
    </div>

    <div class="row">
      <div class="col-sm-12">
        <h5><span style="color: #0000ff;"><b>Critical Care and Personalized or Precision Medicine: Who needs whom? </b></span></h5>
      </div>
      <div class="col-sm-12">
        <p>A new era of personalized and precision medicine (P-Medicine) has arrived with the start of big data analytics as well as computational biology. There are trends emerging for more accurate diagnosis and throughout several assay possibilities. Prediction is the main advantage and computational biology brings with the possibility of developing machine learning algorithms, but requires big data. Answering the question who needs whom, is two-sided of critical care and personalized or precision medicine as well as for medicine and computational biology.</p>
      </div>
      <div class="col-sm-12">
        <p><b>Reference:</b> <a href="http://www.jccjournal.org/article/S0883-9441(17)31674-X/pdf">Brief communication</a></p>
      </div>
    </div>
  </div>
</body>
