<!doctype html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <meta name="description" content="landing page">
  <meta name="author" content="Tariq Khaleeq, Nikos Papadopoulos">
  <link rel="icon" href="../../media/favicon.ico">

  <title>March 2018</title>

  <!-- Bootstrap core CSS -->
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/css/bootstrap.min.css" integrity="sha384-Zug+QiDoJOrZ5t4lssLdxGhVrurbmBWopoEl+M6BdEfwnCJZtKxi1KgxUyJq13dy"
    crossorigin="anonymous">
  <link href="../../style.css" rel="stylesheet" type="text/css" />
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
  <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
  <style>
    body {
      font-family: 'Roboto';
    }
  </style>
</head>

<body>
  <nav class="navbar navbar-expand-md oxford_background top">
    <a class="navbar-brand text-white" href="../../main.html">RSG Germany</a>
    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse"
      aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div class="collapse navbar-collapse" id="navbarCollapse">
      <ul class="navbar-nav ml-auto">
        <li class="nav-item active">
          <a class="nav-link disabled text-white" href="../../main.html">Home
          </a>
        </li>
        <li class="nav-item">
          <a class="nav-link disabled text-white" href="../../events.html">Events</a>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="../../about.html">About
            <span class="sr-only">(current)</span>
          </a>
        </li>
        <li class="nav-item">
          <a class="nav-link disabled text-white" href="../../newsletter.html">Newsletter</a>
        </li>
        <li class="nav-item">
          <a class="nav-link disabled text-white" href="../../contact.html">Contact</a>
        </li>
      </ul>
    </div>

  </nav>
  <!--TODO: add subscription button. Add Footer.
    Probably stylize background. Figure a way to make this dynamic other wise would require bruteforce to update this every month-->
  <div class="container">
    <div class="row">
      <div class="col-sm-12 text-center">
        <br>
        <h1>March 2018</h1>
      </div>
      <div class="col-sm-12">
        <br>
        <h5>Introduction</h5>
      </div>
      <div class="col-sm-12">
        <p>Welcome back again RSG friends! The easter is over and we are going towards the summer, so lets take the advantage
          of these sunny german days to have a look to the May newsletter of the German Regional Student Group from the iSCB
          Student Council. We have decided for this month to select three papers related to population genomics and how this
          field can be useful for our society. In the first we will discuss about what would be the impact in considering
          gender and race/ethnicity in the health systems during the process of diagnosis. The second paper will talk about
          the contribution of genetic variants and negative selection in complex traits such as weight and obesity, finally
          we will see an example of how genomic variations impact the colors of our hairs. See you the next month, stay tuned!
          Yours:</p>

        <em>RGS Germany (ISCB Student Council) – Tommaso Andreani, Ilkay Başak Uysal, Neetika Nath, Nikos Papadopoulos, Yvonne
          Gladbach</em>
        <hr>
      </div>
    </div>
    <div class="row">
      <div class="col-sm-12">
        <br>
        <h5>
          <b>
            <span>Racial and Ethnic Disparities in Cancer Survival: The Contribution of Tumor, Sociodemographic, Institutional,
              and Neighborhood Characteristics
            </span>
          </b>
        </h5>
      </div>
      <div class="col-sm-12">
        <p>Health is wealth! We all have heard this term some point of time in our life. So far this is achieved independently
          by different countries with continuous improvement in the medical care system. However, we find ourselves asking
          this important question: Is our current understanding of medicine good enough for treatment of different race/ethnicity
          or gender? A study conducted by The California Cancer Registry (CCR) of Ellis et al. in 2017, used data about almost
          900.000 patients collected between 2000 and 2013 to study the influence of race/ethnicity, gender, and socioeconomic
          status on cancer survival. Among other factors considered for the study, there were 1) tumour characteristics at
          onset, 2) disease management, 3) treatments, 4) factors that relate to the health care institute where the treatment
          was performed and 5) the sociodemographic and neighbourhood characteristics in which the patients were living.
        </p>

        <p>By applying mediation analysis, the authors quantify the contribution of patient and tumour characteristics to racial/ethnic
          survival disparities and rather than race being the reason for better or worse survival rates, they hypothesized
          that other circumstances, closely related to race, might explain the phenomenon better. The particular question
          asked here would be whether disease management and treatments are possible mediators for cancer mortality in different
          race/ethnicity or gender.
        </p>

        <p>
          Agreeing with conventional wisdom, the study found that early tumour detection is the most effective treatment strategy,
          irrespective of the race/ethnicity or gender. The tumour stage at the time of diagnosis had the largest effect
          on race/ethnicity disparities in survival for breast, prostate, and colorectal cancers. This, in turn, reflects
          other factors that include socioeconomic status, health insurance and access to health care. Race/ethnicity disparities
          in breast cancer survival vary considerably according to tumour subtypes, which was not limited to simply earlier
          detection of cancer. In a single case example, the hormone receptor status was important to investigate survival
          disparities in breast cancer, especially for black women. Without doubt, further detailed information regarding
          clinical treatments that consider race and ethnicity as important variables, could provide important insight into
          the resolution of the differences in cancer survival among single individuals. These studies are very important
          not only for the understanding but to improve our healthcare system and towards personalized medicine.

        </p>

      </div>
      <div class="col-sm-12">
        <p>
          <b>Reference:</b>
          <a href="https://www.cancer-rates.info/ca">Ellis, Libby, et al. “Racial and Ethnic Disparities in Cancer Survival: The Contribution of Tumor, Sociodemographic,
            Institutional, and Neighborhood Characteristics.” Journal of Clinical Oncology 36.1 (2017): 25-33.
          </a>
        </p>
      </div>
    </div>

    <div class="row">
      <div class="col-sm-12">
        <h5>
          <span>
            <b>Signatures of negative selection in the genetic architecture of human complex traits</b>
          </span>
        </h5>
      </div>
      <div class="col-sm-12">
        <p>
          Understand the genetic basis of complex molecular traits such as cancer, obesity or alzheimer has attract a lot of attentions
          in “modern” genetic. The improvement of genomics analysis in terms of cost and time has allowed scientists to test
          the millions of genetic variants of the human genome to specific polygenic traits. It is becoming clear that understanding
          the role of single nucleotide polymorphisms to a particular phenotype is far more complex than what explained by
          the mendelian genetic in which a mutation in one locus is responsible of a given phenotype.
        </p>

        <p>Natural selection on genetic variants, is one of the driving force of when and how an allele is selected or not in
          the next gene pool for a particular phenotype (positive and negative selection respectively). This concept has
          attracted a lot of attention in the last few years in cohort of genome wide association studies. Indeed, it has
          been proposed that natural selection is likely to act simultaneously on many trait-associated variants that have
          pleiotropic effects on fitness. But, what does this exactly mean? In practice, according to the author, specific
          genetic variants along the genome are more prone to undergo to negative selection (selective sweep model). Example
          of those variants are the ones directly involved in a particular disease and that negative selection is likely
          to select against (eliminate) in most gene pool individuals. However, others variants with small effects that are
          typical of complex traits are difficult to estimate whether they are under negative or positive selection.
        </p>

        <p>In order to detect the contribution of negative and positive selection on those variants with small effects on complex
          traits, the authors developed a Bayesian mixed linear model (MLM) method that can simultaneously estimate SNP-based
          heritability, polygenicity, and the joint distribution of effect size and minor allele frequency (MAF) in conventionally
          unrelated individuals. What they found was that out of the 28 traits analyzed, 23 of them are associated with strong
          selection. More important, they were able to report that most of these traits related variants were undergoing
          to negative selection (for example for height and body mass index).
        </p>
        <p>In conclusion, future GWAS based on whole genome sequencing, or imputed data with large sample sizes, are expected
          to discover an increasing number of rare variants in which the sum of them will have a large effect in the phenotypic
          outcome of a given population.
        </p>

      </div>
      <div class="col-sm-12">
        <p>
          <b>Reference:</b>
          <a href="https://www.nature.com/articles/s41588-018-0101-4">Zeng, Jian, et al. “Signatures of negative selection in the genetic architecture of human complex traits.” Nature
            genetics (2018): 1.</a>
        </p>
      </div>
    </div>

    <div class="row">
      <div class="col-sm-12">
        <h5>
          <span>
            <b>Genome-wide association meta-analysis of individuals of European ancestry identifies new loci explaining a substantial
              fraction of hair color variation and heritability
            </b>
          </span>
        </h5>
      </div>
      <div class="col-sm-12">
        <p>This study provides insight to melanin pigment metabolism in humans and how it is explained by single-nucleotide
          polymorphisms associated with hair color.
        </p>

        <p>The results might be useful to better understand molecular human pigmentation, particularly for DNA-based predictions
          with forensic and anthropological applications, and to understand and potentially develop treatment strategies
          for diseases that result from biological impairment of pigmentation.
        </p>

        <p>In this large-scale study, the authors analysed 300.00 people’s genomes. By comparing their genomic profile, they
          discovered 124 genes significantly associated with hair colour; one of which was on the X chromosome and only 13
          of them were novel.
        </p>

        <p>
          A meta-analysis of two GWAS was carried out in two large discovery cohort studies: 157,653 research participants from the
          23andMe, Inc. customer base18 and 133,238 individuals from the UK Biobank (UKBB). The participants self-reported
          their natural hair colour in adulthood. A numerical category system was used for this purpose and each hair color
          category collected (black, dark brown, light brown, red, and blond) was assigned to a numerical values ranging
          from lowest (blond) to highest (black). These codes were used as the outcome variable in linear regression-based
          GWAS analyses. To minimize population admixture and stratification, the analyses were restricted to individuals
          of European ancestry and adjusted for the first ten principal components of the genotype matrix, as well as for
          age and sex.
        </p>

        <p>Hair colour associated single-nucleotide polymorphisms were found to be responsible for 34.6% of red hair, 25% of
          blond hair, and 26% of black hair heritability in the group of people studied. These results confirm the polygenic
          nature of complex phenotypes as known before.
        </p>

        <p>The study findings also provide insights into disease and sex-specific pigmentation. Woman in the study are more
          likely to report their hair blond or red than any other color and three to five times less likely to report black
          hair compared to men. This result shows that sex is associated with hair colour independent of socially driven
          self-reporting bias which is in line with previous findings that are based on objective measurement of hair colour.
          Although hair pigmentation spans a spectrum from very bright (blonde) to very dark (black), the genetic mechanisms
          do not always follow this linear scale, as red hair color often has unique predisposing genetic factors. However,
          this study explains higher portions of heritability than before.
        </p>

        <p>As expected, genes showing the strongest association in the meta-analysis were significantly enriched for several
          Gene Ontology entries, especially pigmentation and melanin biosynthetic and metabolic processes. Moreover, this
          method outperformed one of the pioneer studies that used the HIrisPlex model for predicting eye and hair colour
          from DNA (The HIrisPlex system for simultaneous prediction of hair and eye colour from DNA, Walsh et al. 2013).
        </p>

        <p>The authors used their results to predict black and red hair colours with a high degree of accuracy using a separate
          set of genetic data in which hair colour was known. Although predictions for brown and blond hair colour were less
          accurate, this can potentially help in predicting hair colour based on DNA evidence in forensic investigations.
        </p>
      </div>
      <div class="col-sm-12">
        <p>
          <b>Reference:</b>
          <a href="https://www.nature.com/articles/s41588-018-0100-5"> Pirro G. Hysi, et al. “Genome-wide association meta-analysis of individuals of European ancestry identifies new
            loci explaining a substantial fraction of hair color variation and heritability” Nature genetics(2018).</a>
        </p>
      </div>
    </div>
  </div>
  <!--TODO: Fix footer; add rows properly-->
  <footer class="oxford_background">
    <div class="container-fluid">
      <br>
      <div class="row mt-3">
        <div class="col-4 text-white  ">
          <div class="row">
            <div class="col-lg-4">
              Street no.123
            </div>
            <div class="w-100"></div>
            <div class="col-lg-5 ">
              City Somewhere
            </div>
            <div class="w-100"></div>
            <div class="col-lg-6 ">
              Country Somewhere
            </div>
            <div class="w-100"></div>
          </div>
        </div>
        <div class="col-4 text-white">
          <ul class="mx-auto" style="list-style-type: none; padding: 0; text-align: center">
            <li>
              <a class="text-white" href="../../main.html">Home</a>
            </li>
            <li>
              <a class="text-white" href="../../about.html">About</a>
            </li>
            <li>
              <a class="text-white" href="../../contact.html">Contact</a>
            </li>
          </ul>
        </div>
        <div class="col-4 text-white">
          <address class="mr-4" style="text-align: right">
            Follow us:
            <br>
            <a href="https://twitter.com/GermanyRsg" class="fa fa-twitter"></a>
            <br>
          </address>
        </div>
      </div>
      <div class="row text-white">
        <p class="mx-auto">Trademarked 2018</p>
      </div>
    </div>
  </footer>
  </main>


  <!-- Bootstrap core JavaScript
          ================================================== -->
  <!-- Placed at the end of the document so the pages load faster -->
  <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
    crossorigin="anonymous"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
    crossorigin="anonymous"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/js/bootstrap.min.js" integrity="sha384-a5N7Y/aK3qNeh15eJKGWxsqtnX/wWdSZSKp+81YjTmS15nvnvxKHuzaWwXHDli+4"
    crossorigin="anonymous"></script>
</body>

</html>